src/main.ts

```diff
@@ -123,6 +124,42 @@ export default function main() {
     return res;
   }
 
+  type FeedbackOptions = {
+    featureId: string;
+    sentiment: "like" | "dislike";
+    userId?: string;
+    companyId?: string;
+    comment?: string;
+  };
```

Comment:
This looks identical to the `Feedback` type in `src/types.ts` - perhaps just get rid of this?

Reply:
this one is slightly better. Might do that

<img width="718" alt="Screenshot 2023-01-09 at 14 11 22" src="https://user-images.githubusercontent.com/744/211315788-29f26d9c-b510-4a33-bf88-989cc68d583b.png">